skel/config/config.go

```diff
@@ -1,34 +1,23 @@
-package sonic
+package config
 
 import (
-	"github.com/openware/pkg/database"
 	"github.com/openware/pkg/mngapi"
+	sonicCfg "github.com/openware/pkg/sonic/config"
 )
 
 // Config is the application configuration structure
 type Config struct {
-	Database database.Config `yaml:"database"`
+	Database DatabaseConfig `yaml:"database"`
```

Comment:
Could we still use database.Config here?

Reply:
in the recent `pkg`, the `database` config is removed not sure what's the reason. or just the `database.Config` you want to put it in sonic pkg?